airbyte-integrations/connectors/source-notion/unit_tests/test_streams.py

```diff
@@ -76,3 +77,61 @@ def test_backoff_time(patch_base_class):
     stream = NotionStream(config=MagicMock())
     expected_backoff_time = None
     assert stream.backoff_time(response_mock) == expected_backoff_time
+
+
+def test_users_request_params(patch_base_class):
+    stream = Users(config=MagicMock())
+
+    # No next_page_token. First pull
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": None}
+    expected_params = {"page_size": 100}
+    assert stream.request_params(**inputs) == expected_params
+
+    # When getting pages after the first pull.
+    inputs = {"stream_slice": None, "stream_state": None, "next_page_token": {"next_cursor": "123"}}
+    expected_params = {"start_cursor": "123", "page_size": 100}
+    assert stream.request_params(**inputs) == expected_params
+
+
+def test_user_stream_handles_pagination_correclty(requests_mock):
+    """
+    Test shows that Users stream uses pagination as per Notion API docs.
+    """
+
+    response_body = {
+        "object": "list",
```

Comment:
In the original response from **_Notion_** both **_object_** and **_type_** are part of the **_results_** array. In my opinion It would be better if the tests are kept consistent with the actual implementation
![image](https://user-images.githubusercontent.com/10071532/160299110-2606bb0c-ba3d-4f68-84cc-9a2c737d2858.png)

Reply:
Done in [fda250e](https://github.com/airbytehq/airbyte/pull/11452/commits/fda250ecf078624eff22eb745c3eaa7337c947ce).